src/app/page.tsx

```
@@ -23,30 +34,113 @@ export default function Home() {
-                        {/*<Link href="/game/tic-tac-toe">Start Game</Link>*/}
+            </div>
+            </header>
+            <div className={styles.contentBackground}>
+            {/*Trending Games Section */}
+            <section className="trendingSection">
+                <h2 className={styles.sectionHeading}>Trending Games</h2>
+                {/*Tic-Tac-Toe Card*/}
+                <div className={`${styles.trendingCard} ${styles.cardLeft}`}
+                onClick={()=>handleCardClick("/game/tic-tac-toe")}
+                role="link"
+                tabIndex={0} >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)